backend/src/backend/router.gleam

```diff
@@ -100,7 +101,25 @@ pub fn auth(req: wisp.Request, ctx: Context) -> wisp.Response {
 }
 
 pub fn project(req: wisp.Request, ctx: Context) -> wisp.Response {
-  wisp.response(501)
+  let assert ["api", "v1", "project", ..route] = wisp.path_segments(req)
+
+  case route {
+    [] -> {
+      case req.method {
+        Get -> {
+          wisp.response(501)
+        }
+        Put -> {
```

Comment:
Also make sure to use wisp.required_method() to reject any methods that the api doesn't support